### DISCUSSION TITLE
Index page

### DISCUSSION THREAD

**Discussion by j86schroeder** [alangrainger/share-note#145] [Vote for upcoming features]
Can we add the feature for the index page to be added into the "properties" of any shared note?
<hr>

This feature given that the index page is already created, and shared would automatically show your shared notes on the index page.

<img width="1151" alt="Screenshot 2024-11-30 at 10 42 04 AM" src="https://github.com/user-attachments/assets/9b5d66c1-29d3-4722-ab87-df5a66188615">
<hr>
If this feature could be implemented, then there would be no need for me to reference my index page as you see in my screenshot below:
<img width="1181" alt="Screenshot 2024-11-30 at 10 41 41 AM" src="https://github.com/user-attachments/assets/10912e9a-f4c5-4c49-ab51-be1880e0e228">
<hr>
I would also suggest if this feature can be added, it would be great if we had the option to hide the "index" page from the shared note, and also the option to display or hide any shared note listed on the index page.
<hr>
https://note.sx/roadmap

Thank you!

**Comment by j86schroeder** [comment]
Just to clarify for example, if there was a "property" called "share_index", and you toggle it on in your shared note, it would then automatically populate and show the "index" page right in your specific shared note, that way you can navigate from that shared note to the index page.

<hr>
<img width="1208" alt="Screenshot 2024-11-30 at 11 01 46 AM" src="https://github.com/user-attachments/assets/dca8293a-6cc7-4484-a683-722de740e24b">

**Comment by TangentFoxy** [comment]
The suggested idea of a single index makes it inconvenient for a lot of shared notes with varying topics. For an example of an organization system that would break, I shameless share [my index](https://share.note.sx/gdon3y36).

(Adding the extra links back to an index isn't too much work, and if you really want an automated population of the index page, I would suggest looking at dataview queries since this already supports them, or finding a way to make a script automatically run to generate it when necessary.)
